extractors/pornhub/pornhub.go

```diff
@@ -48,9 +51,28 @@ func (e *extractor) Extract(url string, option types.Options) ([]*types.Data, er
 		return nil, err
 	}
 
+	scripts := utils.MatchAll(html, `<script type="text/javascript">(?s)(.+?)</script>`)
+	var flashvarJS string
+	for _, s := range scripts {
+		for _, js := range s[1:] {
+			if strings.Contains(js, "flashvars_") {
+				flashvarJS = js
+				break
+			}
+		}
+	}
+	vm := otto.New()
+	vm.Run(flashvarJS)
```

Comment:
```suggestion
	vm.Run(flashvarJS) // nolint
```

One more thing, the error return value of `vm.Run` is not checked, this didn't pass golangci-lint's check